ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -245,6 +249,60 @@ def verify_noobaa_diagnostics(self):
                 logger.error("noobaa_diagnostics.tar.gz does not exist")
                 self.files_not_exist.append("noobaa_diagnostics.tar.gz")
 
+    def get_all_paths(self):
+        """
+        Get all paths in must gather dir
+
+        """
+        for root, dirs, files in os.walk(self.root):
+            for name in files + dirs:
+                full_path = os.path.join(root, name)
+                self.full_paths.append(full_path)
+
+    def verify_paths_in_dir(self, paths):
+        """
+        Verify paths exist in must gather directory
+
+        Args:
+            paths (list): list of paths in mg directory for example
+            "/ceph_logs/journal_" exist in /mg_dir/a/b/ceph/ceph_logs/journal_compute-1/log.log
+
+        Returns:
+            list: the paths do not exist in mg dir
+
+        """
+        paths_exist = []
+        for path in paths:
+            status = False
+            for full_path in self.full_paths:
+                if path in full_path:
+                    status = True
+            if not status:
+                paths_exist.append(path)
+        return paths_exist
+
+    def verify_paths_not_in_dir(self, paths):
+        """
+        Verify paths do not exist in must gather directory
+
+        Args:
+            paths (list): list of paths in mg directory for example
+            "/ceph_logs/journal_" exist in /mg_dir/a/b/ceph/ceph_logs/journal_compute-1/log.log
```

Comment:
this should be indented one more to the right

Reply:
@petr-balogh  Can you check my fix? https://github.com/red-hat-storage/ocs-ci/pull/9343/commits/831d9f35e96bacb615c434341acd29c604581af0